draft-ietf-rats-architecture.md

```diff
@@ -860,7 +860,17 @@ trusts a Verifier that can appraise the trustworthiness of
 information about an Attester.  Such trust might come by the Relying
 Party trusting the Verifier (or its public key) directly, or might
 come by trusting an entity (e.g., a Certificate Authority) that is
-in the Verifier's certificate chain.
+in the Verifier's certificate chain.  Such trust
```

Comment:
From RFC 4949

   $ certificate chain
      (D) Synonym for "certification path". (See: trust chain.)

      Deprecated Term: IDOCs SHOULD NOT use this term; it duplicates the
      meaning of a standardized term. Instead, use "certification path".

Reply:
Done